app/src/test/java/org/listenbrainz/android/MockLookupRepository.kt

```diff
@@ -37,7 +37,7 @@ class MockLookupRepository : LookupRepository {
         TODO("Not yet implemented")
     }
 
-    override suspend fun fetchAcoustIDResults(duration: Long, fingerprint: String?): Resource<List<Recording>> {
+    suspend fun fetchAcoustIDResults(duration: Long, fingerprint: String?): Resource<List<Recording>> {
```

Comment:
Please remove this. I think we missed this while the cleanup was being done

Reply:
Done